vision/api/ProductSearch/ImportProductSets.cs

```diff
@@ -0,0 +1,71 @@
+﻿using CommandLine;
+using Google.Cloud.Vision.V1;
+using Google.LongRunning;
+using System;
+
+namespace GoogleCloudSamples
+{
+    [Verb("import_product_set", HelpText = "Import a product set from GCS")]
+    class ImportProductSetOptions : BaseOptions
+    {
+        [Value(2, HelpText = "Google Cloud Storage URI. Target files must be in Product Search CSV format.")]
+        public string GcsUri { get; set; }
+    }
+
+    public class ImportProductSets
+    {
+        public static void RegisterCommands(VerbMap<object> verbMap)
+        {
+            verbMap
+                .Add((ImportProductSetOptions opts) => ImportProductSet(opts));
+        }
+
+        // [START vision_product_search_import_product_images]
+        private static int ImportProductSet(ImportProductSetOptions opts)
+        {
+            var client = ProductSearchClient.Create();
+            var request = new ImportProductSetsRequest
+            {
+                // A resource that represents Google Cloud Platform location.
+                ParentAsLocationName = new LocationName(opts.ProjectID,
+                                                        opts.ComputeRegion),
+
+                // Set the input configuration along with Google Cloud Storage URI
+                InputConfig = new ImportProductSetsInputConfig
+                {
+                    GcsSource = new ImportProductSetsGcsSource
+                    {
+                        CsvFileUri = opts.GcsUri
+                    }
+                }
+            };
+            Operation<ImportProductSetsResponse, BatchOperationMetadata> response = client.ImportProductSets(request);
```

Comment:
Do the other existing LRO operation samples explicitly show the type?

I think we normally do this, which would be more idiomatic IMO (unless you're following an existing standard)

You can browse through some other examples in the repo:
https://github.com/GoogleCloudPlatform/dotnet-docs-samples/search?q=PollUntilCompleted&unscoped_q=PollUntilCompleted

```cs
var operation = client.ImportProductSets(request);

// Wait for operation to complete
var response = operation.PollUntilCompleted();

foreach (var status in response.Result.Statuses)
{
  // ...
}
```

Reply:
Here's the thing ... when I didn't explicitly set the type, the operations returned before they completed.

When I did explicitly set the type, they completed and then returned.

Reply:
... and of course, now that I change back to var it works again.

Thank you!